src/app/utils/bibDetailsUtils.jsx

```diff
@@ -212,6 +212,73 @@ const constructSubjectHeadingsArray = (url = '') => {
     });
 };
 
+/**
+ * isRtl(string)
+ * Returns true (or false) depending on whether the string needs to be read right to left
+ * @param {string} string
+ * @return  {boolean}
+ */
+const isRtl  = (string) => {
+  if (typeof string !== 'string') { return null }
```

Comment:
Should this be `false` to be consistent with the comment above?

Reply:
Yep, fixed